modules/internal/DefaultSparse.chpl

```diff
@@ -269,6 +293,28 @@ module DefaultSparse {
       for e in data[1..dom.nnz] do yield e;
     }
 
+    iter these(param tag: iterKind) ref where tag == iterKind.standalone {
```

Comment:
Could this iter forward to the dom iter somehow? 

If it were serial it seems like you could do:
```chapel
for i in dom do
  yield data[i];
```

What would 

```chapel
iter these(param tag: iterKind) ref where tag == iterKind.standalone {
  forall i in dom do 
    yield 
}
```

turn into? Would this even work, and if it did would it add extra overhead?